src/app/pages/AccountPage.jsx

```diff
@@ -36,16 +37,15 @@ const AccountPage = (props) => {
 
   const [isLoading, setIsLoading] = useState(true);
   const [itemToCancel, setItemToCancel] = useState(null);
+  const [accountPageModal, setAccountPageModal] = useState(false);
```

Comment:
can this be `displayModal`, `setDisplayModal`?

Reply:
sure. Changed.